UDV-Core/src/Extensions/Contribute/View/DocumentCreationWindow.js

```diff
@@ -147,6 +157,22 @@ export class DocumentCreationWindow extends AbstractDocumentWindow {
     fileReader.readAsDataURL(this.docImageElement.files[0]);
   }
 
+  _enterEditMode() {
+    this.controls.rotateSpeed = 1.5;
+    this.controls.zoomInFactor = 0.04;
+    this.controls.zoomOutFactor = 0.04;
+    this.controls.maxPanSpeed = 5.0;
+    this.controls.minPanSpeed = 0.01;
+  }
+
+  _exitEditMode() {
+    this.controls.rotateSpeed = 2.0;
+    this.controls.zoomInFactor = 0.25;
+    this.controls.zoomOutFactor = 0.4;
+    this.controls.maxPanSpeed = 15.0;
+    this.controls.minPanSpeed = 0.05;
+  }
+
```

Comment:
Could you comment these functions please ?

Reply:
Done (also a bit refactored)